udata/tests/api/test_user_api.py

```diff
@@ -170,3 +170,22 @@ def test_suggest_users_api_by_id(self):
 
         suggestion = response.json[0]
         self.assertEqual(suggestion['id'], str(user.id))
+
+    def test_users(self):
+        '''It should provide a list of users'''
+        with self.autoindex():
+            user = UserFactory(
+                about='* Title 1\n* Title 2',
+                website='http://www.datagouv.fr/user',
+                avatar_url='http://www.datagouv.fr/avatar',
+                metrics={'datasets': 10})
+        response = self.get(url_for('api.users'))
+        self.assert200(response)
+        json = response.json['data'][0]
```

Comment:
I'll test that `json['data']` length is 1. It seems obvious but when it's not the case anymore, the error will be more explicit.